### PR TITLE
Link back to parameters

### PR DESCRIPTION
**Pull request by eeliana**

### CODE REVIEW

**Comment by lingshanng** [comment]
mention parameters after the command format?

File: docs/UserGuide.md [hunk 1]
```diff
@@ -386,9 +388,11 @@ Examples:
 
 #### Editing a student: `edit`
 
-Edits an existing student in TAB.
+Edits an existing student in TAB.<br>
+
+Format: `edit INDEX [n/NAME] [a/ADDRESS] [p/PHONE] [e/EMAIL] [pp/PARENT_PHONE_NUMBER] [pe/PARENT_EMAIL] [sch/SCHOOL] [stream/ACAD_STREAM] [lvl/ACAD_LEVEL] [r/REMARK] [t/TAG]…​`<br>
 
-Format: `edit INDEX [n/NAME] [a/ADDRESS] [p/PHONE] [e/EMAIL] [pp/PARENT_PHONE_NUMBER] [pe/PARENT_EMAIL] [sch/SCHOOL] [stream/ACAD_STREAM] [lvl/ACAD_LEVEL] [r/REMARK] [t/TAG]…​`
+Available fields are as described in [Student Parameters](#student-parameters).
```
Comment (Xiaoyunnn) [inline docs/UserGuide.md:395]:
```suggestion
Available fields are as described in [Student parameters](#student-parameters).
```
minor detail but just to be consistent with lesson param

Comment (Xiaoyunnn) [inline docs/UserGuide.md:395]:
```suggestion
Available fields are as described in [Student parameters](#student-parameters).
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -705,9 +710,11 @@ Do look forward to these features!
 
 #### Adding a lesson: `ladd`
 
-Adds a lesson to the specified student in TAB, provided you do not have any other lessons scheduled at that time.
+Adds a lesson to the specified student in TAB, provided you do not have any other lessons scheduled at that time.<br>
+
+Format: `ladd INDEX [recurring/END_DATE] date/START_DATE time/TIME_RANGE subject/SUBJECT rates/LESSON_RATES [f/OUTSTANDING_FEES] [hw/HOMEWORK]…​`<br>
 
-Format: `ladd INDEX [recurring/END_DATE] date/START_DATE time/TIME_RANGE subject/SUBJECT rates/LESSON_RATES [f/OUTSTANDING_FEES] [hw/HOMEWORK]…​`
+Available fields are as described in [Lesson Parameters](#lesson-parameters).
```
Comment (Xiaoyunnn) [inline docs/UserGuide.md:717]:
```suggestion
Available fields are as described in [Lesson parameters](#lesson-parameters).
```

**Review by Xiaoyunnn — COMMENTED** [review]
minor detail: the `parameters` for both table headers are small letter

**Review by Xiaoyunnn — APPROVED** [review]
lgtm

**Review by Chesterwongz — APPROVED** [review]

**Pull request merged**